src/components/AdminOrderItem/style.less

```diff
@@ -0,0 +1,34 @@
+@import '../../styles/vars.less';
+
+.order-item {
+  max-width: 100%;
+  min-width: 100%;
+
```

Comment:
why not just `width: 100%`

Reply:
Whoop! That's pretty funny to catch. Fixed in 1dee5fc.